### DISCUSSION TITLE
수환님 버그 있어욤

### DISCUSSION THREAD

**Discussion by leedh2004** [SuperNova911/Gardener-for-GitHub#19] [Bug]
**Describe the bug**
메인화면 활동하고 요약이 글자가 겹쳐요 의도는 아닐거같아서
이거 서버 직접 만든건가요 혹시?

**To Reproduce**
메인화면

**Expected behavior**
활동에 백그라운드 있어야 할듯

**User information**
- Device: iPhone14,2
- OS: iOS 15.1.1
- App Version: 1.0.0 (21)

**Additional context**

**Comment by SuperNova911** [comment]
그냥 SwiftUI에서 제공하는 타이틀 썼던거같은데, iOS 15에선 버그가 생기더라구요;;

**Comment by SuperNova911** [comment]
엑코 13으로 빌드 새로 때리니까 고쳐짐;
